libursa/src/signatures/ed25519.rs

```diff
@@ -34,6 +34,36 @@ impl Ed25519Sha512 {
             ))),
         }
     }
+
```

Comment:
Please add comments here to document the purpose of the sign_key_to_key_exchange function and how to use it.